### PR TITLE
Add some information in the changelog, add the updated versions

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 1]
```diff
@@ -8,6 +8,189 @@ Version 2.7.0
 Information to know before starting the upgrade
 ===============================================
 
+The Upgrade to the Web Components started, the authentication and the LIDAR profile are migrated.
+In the desktop alt we introduce a canvas component who manage the application layout and plug the
+components throw slots.
```
Comment (llienher) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:13]:
```suggestion
components through slots.
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 2]
```diff
@@ -8,6 +8,189 @@ Version 2.7.0
...
+In ngeo the sources files of gmf are moved in ngeo src folder.
```
Comment (llienher) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:14]:
```suggestion
In ngeo, the sources files of GMF are moved in ngeo src folder.
```
:-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt [hunk 3]
```diff
@@ -8,6 +8,189 @@ Version 2.7.0
 Information to know before starting the upgrade
 ===============================================
 
+The Upgrade to the Web Components started, the authentication and the LIDAR profile are migrated.
+In the desktop alt we introduce a canvas component who manage the application layout and plug the
```
Comment (llienher) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt:12]:
```suggestion
In the desktop_alt we introduce a canvas component who manage the application layout and plug the
```

**Review by llienher — APPROVED** [review]

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt (hunk 2 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt]: It's the folder => changed to `contribs/gmf/src`

Comment (llienher) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt]: I know, I was just doing some syntax checking, but maybe add it to the sentence if it's better to understand.

**Pull request merged**